addons/field-guide/index.js

```diff
@@ -107,11 +107,12 @@ module.exports = {
   },
 
   // TODO! not sure how to test if this logic below works
-  urlsForPrember(distDir) {
-    const flatPageListJson = fs.readJsonSync(`${distDir}/toc.json`);
-    // TODO is there a way to have this list generated automatically (or exported) from the routes (`website/app/router.js`)?
-    const staticURLs = ['/', 'about', 'foundations', 'components', 'patterns'];
-    const docsURLs = flatPageListJson.map((page) => page.pageURL);
-    return [...staticURLs, ...docsURLs];
-  },
+  // https://github.com/ef4/prember#using-prember-from-an-addon
+  // urlsForPrember(distDir) {
```

Comment:
at least for now I thought this would be useful to leave behind for future us if we decide to revisit this but I have no strong opinions on it